hamster_gtk/preferences/preferences_dialog.py

```diff
@@ -76,6 +78,8 @@ def __init__(self, parent, app, initial, *args, **kwargs):
             (_('Miscellaneous'), LabelledWidgetsGrid(collections.OrderedDict([
                 ('autocomplete_activities_offset', (_("Autocomplete Activities Offset"),
                     HamsterSpinButton(SimpleAdjustment(0, GObject.G_MAXDOUBLE, 1)))),
+                ('autocomplete_split_activity', (_("Autocomplete Split Activity"),
```

Comment:
“Autocomplete activities and categories separately” would be probably more meaningful.